lib/big_brother/cluster.rb

```diff
@@ -11,7 +11,7 @@ module Type
     def initialize(name, attributes = {})
       @name = name
       @fwmark = attributes[:fwmark]
-      @cluster_mode = attributes.fetch(:backend_mode, Type::Default)
+      @cluster_mode = @backend_mode = attributes.fetch(:backend_mode, Type::Default)
```

Comment:
@ssgelm I have a few notes to follow up on this and the use of the variable `multi_datacenter`. There's an opportunity for a refactoring there.